thirdeye/thirdeye-frontend/app/pods/custom/metrics-table-offset/template.hbs

```diff
@@ -1,6 +1,7 @@
-<p class="metrics-table__baseline-score">
-  {{get record (concat column.propertyName ".score")}}
-</p>
-<p class="metrics-table--{{record.wo1w.direction}}">
+<span class="metrics-table__change--{{get record (concat column.propertyName ".direction")}}">
+  {{get record (concat column.propertyName ".change")}}
+</span>
+<br/>
+<span class="metrics-table__value">
   {{get record (concat column.propertyName ".value")}}
-</p>
+</span>
```

Comment:
Since these are meant to be displayed in 2 lines, it's more semantic to have `<p>` tags instead of `<span>` because `span` is for inline texts.